replicate-to-sql-with-generic-events/features/step_definitions/steps.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const { Given, Then } = require('cucumber');
+const { Kuzzle, WebSocket } = require('kuzzle-sdk');
+const { spawn } = require('nexpect');
+const { PostgresWrapper } = require('../../lib/postgres');
+
+const localConfig = {
+  password: 'password',
+  database: 'nyc_open_data',
+  user: 'my_user',
+  port: 5432
+};
+const pluginPath = 'plugins/enabled/replicate-to-sql-with-generic-events/scripts';
+
+const options = { env: {'COMPOSE_INTERACTIVE_NO_CLI': 1 }, stream: 'all' };
+
+Given(/A Kuzzle stack with Postgres running/, async function() {
+  this.kuzzle = new Kuzzle(new WebSocket('localhost'));
+  this.postgres = new PostgresWrapper(localConfig);
+  await this.kuzzle.connect();
+  this.pool = await this.postgres.connect();
+});
+
+Then(/I can load the test data into Kuzzle/, function(done) {
+  const scriptPath = `${pluginPath}/import-data.js`;
+  spawn('docker-compose', ['exec', 'kuzzle', 'node', scriptPath], options)
+    .run(function(err, stdout, exitcode) {
+      if (err || exitcode === 1) {
+        throw new Error('failed to execute test');
```

Comment:
> BeforeAll / AfterAll Like hooks and steps, these can be synchronous, accept a callback, or return a promise. Unlike Before / After these methods will not have a world instance as this . This is because each scenario gets its own world instance and these hooks run before / after all scenarios.

Since they cannot access to this, they cannot close/disconnect process created inside the Given hook leading to tests never end.

However i mooved the After outside the BeforeAll hook. 

Has mentionned before this need to be fixed in the boilerplate too